site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -23,35 +23,39 @@ type: docs
 weight: 600
 ---
 
-The default server configuration is intended for development and testing. When you deploy Polaris in production,
-review and apply the following checklist:
+The default server configuration is intended for development and testing. When you deploy Polaris in production, review
+and apply the following checklist:
+
 - [ ] Configure OAuth2 keys
 - [ ] Enforce realm header validation (`require-header=true`)
 - [ ] Use a durable metastore (JDBC + PostgreSQL)
 - [ ] Bootstrap valid realms in the metastore
 - [ ] Disable local FILE storage
+- [ ] Polaris Server Header
+- [ ] Upgrade considerations
+- [ ] Cloud Storage Specific Configuration
 
 ### Configure OAuth2
 
-Polaris authentication requires specifying a token broker factory type. Two implementations are
-supported out of the box:
+Polaris authentication requires specifying a token broker factory type. Two implementations are supported out of the
```

Comment:
nit: I'd prefer not to change the formatting on existing lines when the text itself does not change. It complicates reviews and skews line authorship attribution.